src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+	private static final String DELIMITERS = ",|:";
+	private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+	private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile(CUSTOM_DELIMITER);
+
+	public static int splitAndSum(String input) {
+		if (isNullOrBlank(input)) {
+			return 0;
+		}
+		return sumNumbers(removeDelimiter(input));
+	}
+
+	private static String[] removeDelimiter(String input) {
+		Matcher m = CUSTOM_DELIMITER_PATTERN.matcher(input);
+		if (m.find()) {
+			String customDelimiter = m.group(1);
+			return m.group(2).split(customDelimiter);
+		}
+		return input.split(DELIMITERS);
+	}
+
+	private static boolean isNullOrBlank(String input) {
+		return input == null || input.isEmpty();
+	}
+
+	private static int sumNumbers(String[] numbers) {
+		int sum = 0;
+		for (String number : numbers) {
+			int num = Integer.parseInt(number);
+			isNumNegative(num);
+			sum += num;
+		}
+		return sum;
+	}
+
+	private static void isNumNegative(int num) {
```

Comment:
```suggestion
	private static void validatePositive(int num) {
```
`isXXX`는 boolean 타입을 반환할 때 주로 사용합니다.
주로 if 괄호 안에 들어가는 조건에 사용하면 좋을 것 같아요

지금은 음수인지 판단하고 예외를 던지고 있기 때문에 validate 같은 이름을 사용하는 것이 더 적절한 것 같네요